service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
 the service.
 
 ### Run the application (from IntelliiJ)
- 
-Ensure the Spring Boot is started with local profile 
-(add environment variable spring.profiles.active=local when starting the main class).
 
-Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ 
-starts the application). 
+Ensure the Spring Boot is started with local, user-mappings and the feature-toggle profiles
+(add environment variable spring.profiles.active=feature-toggle,local,user-mappings when starting the main class).
 
-Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
-http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching 
-locally running application. 
+Configure the notify.api_key if necessary (you can pass them as environment variables when IntelliJ
+starts the application).
 
+Ensure the variable CCD_DEF_CASE_SERVICE_BASE_URL bin/configurer/utils/fpl-process-definition.sh is set to
+http://docker.for.mac.localhost:4000 - this will use the host machine for CCD, reaching
+locally running application.
+
+## Config
+
+Services uses three sources of configuration for the placeholders:
```

Comment:
What exactly is Services here? Is it our service, in which case it should be singular and read more like "The service uses ...", or is it external services that we use to build and deploy our service, in which case it should probably be more like "External services use ..."

Reply:
changed to:
The FPL Service (as other HMCTS Reform services)